ideascube/library/tests/test_migrations.py

```diff
@@ -20,6 +20,126 @@
 }
 
 
+@migration_test(
+    migrate_from=[
+        ('library', '0007_auto_20161007_1220'),
+        ('monitoring', '0004_auto_20161007_1240'),
+        ('taggit', '0002_auto_20150616_2121'),
+    ], migrate_to=[('library', '0008_library_stock')])
+def test_library_to_monitoring_with_duplicated_data(migration):
```

Comment:
> Please use more explicit variable name than book, item, specimen when you're testing that migration had run correctly.

Like what?

Since those variables are reused, I don't really see what could be more explicit than the type being tested.

> Please test the books in the same order you've create them (this migration is far complex enough without that)

I'm testing them in the order I did, because I need to `order_by` them to fix the order.

So I've tested them in the order returned by `order_by`.

Do you suggest I test them like so?

```python
# First, the book without specimens
book = books[1]
...

# Next, the book with two specimens
book = books[0]
...

# And now, the book with specimens and relation to the stock
book = books[2]
...
```

I don't mind, just trying to understand if that's what you're asking for.